org.eclipse.xtext.testing/META-INF/MANIFEST.MF

```diff
@@ -28,6 +28,7 @@ Require-Bundle: org.eclipse.xtext;visibility:=reexport,
  org.eclipse.xtend.lib
 Import-Package: org.apache.log4j;version="1.2.15",
  org.apache.log4j.spi;version="1.2.15",
+ org.junit.jupiter.api;version="[5.0.0,6.0.0)";resolution:=optional,
```

Comment:
Interesting find. Why did it even compile without the import?

Reply:
Cause gradle does not check manifest

Reply:
Of course... why would we compile an OSGI module with a tools that understands OSGI manifests ...